include/gatekeeper_mailbox.h

```diff
@@ -19,14 +19,10 @@
 #ifndef _GATEKEEPER_MAILBOX_H_
 #define _GATEKEEPER_MAILBOX_H_
 
-/*
- * TODO Remove this preprocessing directive when there are
- * are calls to these functions; for now, the mailbox is
- * just an example of how to set up a library.
- */
-#if 0
-void create_mailbox(void);
-void destroy_mailbox(void);
-#endif
+#include <rte_ring.h>
+
+struct rte_ring *create_mailbox(char *ring_name,
```

Comment:
`create_mailbox()` should create a `struct rte_ring` and the command memory poll. Mailbox should be a struct type.